README.md

```diff
@@ -13,44 +13,115 @@ View the [list of supported APIs and Services](https://cloud.google.com/php/docs
+place depending on your system.
+
+Windows:
+```
+%APPDATA%\gcloud\application_default_credentials.json
+```
+
+Linux and MacOS:
+```
+$HOME/.config/gcloud/application_default_credentials.json
 ```
 
-You can then include the autoloader and create your client:
+To read more about Authentication, see [AUTHENTICATION.md](AUTHENTICATION.md)
+
+### Instantiating the client
+Once we have the `application_default_credentials.json` that we created on the previous step now we
+can instantiate a client which internally using the Google Auth library will take that file and use
+it to authenticate your requests:
+
+**Note**: You can install the Google Translate client library by running `composer install google/cloud-translate`.
+See the [README.md](README.md) for further instructions, or browse
+[the complete list of packages](https://cloud.google.com/php/docs/reference/).
 
 ```php
-require 'vendor/autoload.php';
+require_once 'vendor/autoload.php';
 
-use Google\Cloud\Storage\StorageClient;
+use Google\Cloud\Translate\V3\Client\TranslationServiceClient;
+use Google\Cloud\Translate\V3\TranslateTextRequest;
+
+// Instantiating the client gathers the credentials from the `application_default_credentials.json`
+$client = new TranslationServiceClient([]);
 
-$storage = new StorageClient();
+$request = new TranslateTextRequest();
+$request->setParent('projects/<YOUR_PROJECT_ID>');
+$request->setTargetLanguageCode('en-US');
+$request->setContents(['こんにちは']);
 
-$bucket = $storage->bucket('my_bucket');
+// The request will contain the authentication token based on the default credentials file
+$response = $client->translateText($request);
 
-// Upload a file to the bucket.
-$bucket->upload(
-    fopen('/data/file.txt', 'r')
-);
+var_dump($response->getTranslations()[0]);
+// {
+//     ["translatedText"]=>
+//     string(5) "Hello"
+//     ["detectedLanguageCode"]=>
+//     string(2) "ja"
+// }
 
-// Download and store an object from the bucket locally.
-$object = $bucket->object('file_backup.txt');
-$object->downloadToFile('/data/file_backup.txt');
 ```
+This quicksetup is built with local development in mind, the steps for deploying your project are
+different depending on the environment you use. Here we provide some basic instruction in how to get
+started with deployment of your project:
+
+### GCP deployment:
+* Use the metadata server instead of a credentials file.
+    - More information about the usage of the metadata server
+      [here](https://cloud.google.com/compute/docs/metadata/overview)
+* Instantiate a client.
+
+### Server deployment:
+* Generate a credentials file.
+    - We recommend using a Service Account instead of the personal account credentials. More
+      information [here](https://cloud.google.com/iam/docs/service-accounts-create)
+* Set the GOOGLE_APPLICATION_CREDENTIALS environment variable pointing to your credentials file.
+* Instantiate a client.
```

Comment:
This section needs to be reworked. I would simplfy it by saying something like this instead:

````suggestion
different depending on the environment you use. 

 * For applications running on Google Cloud Platform (for example, in Google Compute Engine, Google App Engine, Google Kubernetes Engine, and Google Cloud Functions), applications are automatically authenticated using the Metadata Server. For more information, see [About VM Metadata](https://cloud.google.com/compute/docs/metadata/overview)
 * For applications running elsewhere, it is recommended to [create a Service Account](https://cloud.google.com/iam/docs/service-accounts-create) and enable [Workload Identity Federation](https://cloud.google.com/iam/docs/workload-identity-federation).
 * It is also possible to authenticate using [Service Account Keys](https://cloud.google.com/iam/docs/migrate-from-service-account-keys). Set the `GOOGLE_APPLICATION_CREDENTIALS` environment variable pointing to your credentials file.

````